packages/start/routes.js

```diff
@@ -108,9 +109,9 @@ export function stringifyRoutes(routes) {
           i =>
             `{\n${[
               /.data.(js|ts)$/.test(i.dataSrc ?? "")
-                ? `data: ${addImport(process.cwd() + "/" + i.dataSrc)}`
+                ? `data: ${addImport(path.posix.resolve(i.dataSrc))}`
```

Comment:
Does this resolve relative to process.cwd() or is configurable?

Reply:
yes `path.resolve` uses `process.cwd` as default if it's not given an absolute path as a base: https://nodejs.org/api/path.html#pathresolvepaths

> If, after processing all given path segments, an absolute path has not yet been generated, the current working directory is used.